packages/react/README.md

```diff
@@ -1,46 +1,31 @@
-# Getting Started with Create React App
+# DevExtreme UI Template Gallery
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This repository includes responsive UI Templates for the most popular UI/UX patterns in web LOB applications.
```

Comment:
README should be specific for each framework.

E.g. 
We can say that react templates were created based on [devextreme react app template](https://js.devexpress.com/Documentation/Guide/React_Components/Application_Template/). 
And it's ok to describe useful commands that can be executed in this folder (`npm start`, `npm test` etc). To execute `npm install cd packages/react` you are expected to be in the root folder.